pkg/config/config.go

```diff
@@ -534,7 +535,8 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("histogram_copy_to_distribution", false)
 	config.BindEnvAndSetDefault("histogram_copy_to_distribution_prefix", "")
 
-	config.BindEnv("api_key") //nolint:errcheck
+	config.BindEnv("api_key")      //nolint:errcheck
+	config.BindEnv("api_key_file") //nolint:errcheck
```

Comment:
Given the [possible error](https://sourcegraph.com/github.com/spf13/viper/-/blob/viper.go#L1012-1030) I think you can explicitly ingore the error and get rid of the `//nolint:errcheck`.

Reply:
The `nolint:errcheck` hint for `BindEnv(…)` is the way it is done everywhere else in the same function.
We could change it, but everywhere. So, it might deserve a dedicated PR.